app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -101,21 +105,22 @@ def prepare_for_evss!
       Rails.logger.error "Contention Classification failed #{e.message}.", backtrace: e.backtrace
     end
 
-    measure_ep_merge_stats if disabilities.count == 1 && increase_or_new? && classification_updated
+    prepare_for_ep_merge! if disabilities.count == 1 && increase_only? && classification_updated
```

Comment:
The EP400 merge effort is one step behind Contention Classification -- they've recently expanded to **new** single-issue claims, whereas EP400 merge will start with only single-issue claims for **increase**.